enhanced-resolve/lib/Storage.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="node" />
+import { CommonFileSystemMethod } from './common-types';
+
+declare class Storage {
+    duration: number;
+    running: Object;
+    data: Object;
```

Comment:
Values from `data` are given to `Function.apply` / `Tapable.Plugin.apply`, so its type should be `{ [name: string]: any[] }`.